awx/main/scheduler/__init__.py

```diff
@@ -2,3 +2,4 @@
 #
 
 from awx.main.scheduler.task_manager import TaskManager # noqa
```

Comment:
It looks like we want `TaskManager` and `PodManager` part of the public api for `awx.main.scheduler`. You can set the names of these using [the module's `__all__` attribute](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces):

```python
__all__ = ['TaskManager', 'PodManager']
```

You'll see this used other places in awx as well, like [credentials](https://github.com/ansible/awx/blob/devel/awx/main/models/credential/__init__.py#L45). Also, by declaring the names with __all__ you should no longer need to turn off the lint checks on the import statements.